onnxruntime/core/platform/posix/env.cc

```diff
@@ -194,10 +194,16 @@ class PosixEnv : public Env {
     return std::thread::hardware_concurrency();
   }
 
-  std::vector<size_t> GetThreadAffinityMasks() const override {
-    std::vector<size_t> ret(std::thread::hardware_concurrency() / 2);
-    std::iota(ret.begin(), ret.end(), 0);
-    return ret;
+  size_t GetDefaultThreadpoolSetting(std::vector<size_t>& affinity) const override {
+    affinity.clear();
+    affinity.resize(std::thread::hardware_concurrency()>>1);
```

Comment:
it is the number of logic processors divided by two - onnxruntime default threading behavior defined since earlier releases. 
std::thread::hardware_concurrency() returns number of logic processors.